projects/numpy.cmake

```diff
@@ -1,27 +1,10 @@
-
-if(BUILD_OS_LINUX)
-    # Fortran compiler is needed for OpenBLAS, but it does no check whether it is available.
-    enable_language(Fortran)
-    set(openblas_options DYNAMIC_ARCH=1 NO_STATIC=1)
-
-    ExternalProject_Add(OpenBLAS
```

Comment:
Note that I originally included this because I consider this as "one unit" that describes a single dependency of Cura. I do not necessarily mind splitting it up, but that was the thought behind it.